langwatch/src/hooks/useOrganizationTeamProject.ts

```diff
@@ -278,7 +278,7 @@ export const useOrganizationTeamProject = (
     isRefetching: organizations.isRefetching,
     organizations: organizations.data,
     organization,
-    team,
+    team, // The team associated with the current project
```

Comment:
I'm surprised this needs a comment. Consider renaming the variable instead of using a comment.

Reply:
Ah, this was left over because I at one point put the redaction code in here too and then you had two teams running around. I moved that out, but this remains, will remove, thanks!